src/clean.c

```diff
@@ -168,7 +209,7 @@ static int	create_and_add_node(char *start, size_t length, t_list **tokens)
 	return (0);
 }
 
-static int	process_token(char **current_pos, t_list **tokens)
+static int	process_token(char **current_pos, t_list **tokens) //PROBLEMAS AL SEPARAR COMILLAS EN echo hola'mundo' WIP
```

Comment:
this is correct. `hola'mundo'` is a single argument, as there are no spaces in between the quotes. I can showcase examples later in the meeting